src/seedu/addressbook/data/person/UnitNumber.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.data.person;
+
+public class UnitNumber{
+	private String unitNumber;
```

Comment:
You can implement a method to validate the format of the unit number. e.g. must be numbers.